### DISCUSSION TITLE
Failure LRGS Status in Real time

### DISCUSSION THREAD

**Discussion by johnoeo** [opendcs/opendcs#272] [General]
Hi community.
Does anyone know how to solve the update failure of the last oldest message that must be updated in the "LRGS Status in Real time" window?

![Failure_LRGS-Status-Real-time](https://github.com/opendcs/opendcs/assets/47491667/8351a4ae-4490-4aaf-aada-db6fae8a444e)

Regards

**Comment by MikeNeilson** [comment]
I suspect this is a bug, I just looked at one of my setups and it does have a valid time. However I don't have an empty row of "GOES DRGS" so it's possible that whatever fills that value is going through and requesting the last message from each source and that's a default value.

I'll have to check through the code to really figure that out though.
